app/build.gradle

```diff
@@ -331,7 +331,7 @@ dependencies {
     androidTestImplementation Dependencies.fastlane
     implementation Dependencies.falcon // Required by fastlane
 
-    androidTestImplementation Dependencies.espresso_contrib, {
+    androidTestImplementation 'androidx.test.espresso:espresso-contrib:3.1.0-alpha4', {
```

Comment:
We could have kept the `espresso_contrib` variable and reverted the version, but this is a safe alternative too.

Reply:
I would have needed to add an additional change to Dependencies.kt in that case, though. I figured since this is intended to be a temporary workaround it's a bit tidier to change back later.

Reply:
Good point, and we don't have to rebuild the toolchain task this way too.